docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,400 @@
+
+ARG APISIX_VERSION=2.13.1
+LABEL apisix_version="${APISIX_VERSION}"
+
+ARG ENABLE_PROXY
+RUN set -x \
+    && (test "${ENABLE_PROXY}" != "true" || /bin/sed -i 's,http://dl-cdn.alpinelinux.org,https://mirrors.aliyun.com,g' /etc/apk/repositories) \
+    && apk add --no-cache --virtual .builddeps \
+        build-base \
+        automake \
+        autoconf \
+        make \
+        libtool \
+        pkgconfig \
+        cmake \
+        unzip \
+        curl \
+        openssl \
+        git \
+        openldap-dev \
+    && luarocks install https://github.com/apache/apisix/raw/master/rockspec/apisix-${APISIX_VERSION}-0.rockspec --tree=/usr/local/apisix/deps PCRE_DIR=/usr/local/openresty/pcre \
+    && cp -v /usr/local/apisix/deps/lib/luarocks/rocks-5.1/apisix/${APISIX_VERSION}-0/bin/apisix /usr/bin/ \
+    && (function ver_lt { [ "$1" = "$2" ] && return 1 || [ "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]; };  if [ "$APISIX_VERSION" = "master" ] || ver_lt 2.2.0 $APISIX_VERSION; then echo 'use shell ';else bin='#! /usr/local/openresty/luajit/bin/luajit\npackage.path = "/usr/local/apisix/?.lua;" .. package.path'; sed -i "1s@.*@$bin@" /usr/bin/apisix ; fi;) \
+    && mv /usr/local/apisix/deps/share/lua/5.1/apisix /usr/local/apisix \
+    && apk del .builddeps \
+    && apk add --no-cache \
+        bash \
+        curl \
+        libstdc++ \
+        openldap \
+        tzdata \
+    # forward request and error logs to docker log collector
+    && ln -sf /dev/stdout /usr/local/apisix/logs/access.log \
+    && ln -sf /dev/stderr /usr/local/apisix/logs/error.log
+
+WORKDIR /usr/local/apisix
+
+ENV PATH=$PATH:/usr/local/openresty/luajit/bin:/usr/local/openresty/nginx/sbin:/usr/local/openresty/bin
+
+EXPOSE 9080 9443
+
+CMD ["sh", "-c", "/usr/bin/apisix init && /usr/bin/apisix init_etcd && /usr/local/openresty/bin/openresty -p /usr/local/apisix -g 'daemon off;'"]
+
+STOPSIGNAL SIGQUIT
+```
+
+This Dockerfile configuration document, from this [link](https://github.com/apache/apisix-docker/blob/master/alpine/Dockerfile). The only changes I made were as follows:
+
+```bash
+ARG ENABLE_PROXY=false
+
+# Build Apache APISIX
+FROM api7/apisix-base:1.19.9.1.5
+
+ADD ./apisix-go-plugin-runner /usr/local/apisix-go-plugin-runner
+
+ARG APISIX_VERSION=2.13.1
+LABEL apisix_version="${APISIX_VERSION}"
+
+ARG ENABLE_PROXY
+
+```
+
+Package all the `/apisix-go-plugin-runner` files in the `/home/chever/api7/cloud_native/tasks/plugin-runner` directory into a Docker image. Note down the location of the executable `apisix-go-plugin-runner/cmd/go-runner/go-runner` and the location of the `/usr/local/apisix-go-plugin-runner` directory in the Dockerfile above to get the final location of the executable in the Docker image is located as follows.
+
+```bash
+/usr/local/apisix-go-plugin-runner/cmd/go-runner/go-runner
+```
+
+Please make a note of this address. We will use it in the rest of the configuration.
```

Comment:
Why not just base on the APISIX image?

Reply:
waiting for more suggestions~

Reply:
Based on APISIX image will be simpler and more efficient